### PR TITLE
Heroku Migration Guide

### PR DESCRIPTION
**Pull request by forana**

### CODE REVIEW

File: guides/migrating-from-heroku.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
+---
+title: Migrating to Stratum from Heroku
+category: guide
+---
+
+This guide covers the migration of an application that was developed for Heroku to [Stratum](https://catalyze.io/stratum).
+
+## Prerequisites
+
+This guide assumes that you already have an application set up and working - for examples, this will use the application dscribed in the [Node+Mongo guide](/stratum/articles/guides/node-mongo).
```
Comment (anthonypleshek) [inline guides/migrating-from-heroku.md]: described*

File: guides/migrating-from-heroku.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+The other prerequisite is that you're already talked with our Sales team and worked with Catalyze to get an environment provisioned and ready for you, including all of the databases and caches that your application needs. To get that process started, send us a message at [sales@catalyze.io](mailto:sales@catalyze.io).
```
Comment (anthonypleshek) [inline guides/migrating-from-heroku.md]: you've*

**Comment by anthonypleshek** [comment]
LGTM

**Comment by forana** [comment]
changed LGTM as well

**Pull request merged**